docsite/rst/developing_modules.rst

```diff
@@ -238,43 +243,52 @@ Rather than mention these here, the best way to learn is to read some of the `so
 
 The 'group' and 'user' modules are reasonably non-trivial and showcase what this looks like.
 
-Key parts include always ending the module file with::
+Key parts include always importing the boilerplate code from
+:mod:`ansible.module_utils.basic` like this::
 
-    from ansible.module_utils.basic import *
+    from ansible.module_utils.basic import AnsibleModule
     if __name__ == '__main__':
```

Comment:
Is it worth adding a notes to say for new modules this is required?

I've seen a lot of new modules still being created without this `if __name__` conditional